frontend/elements/src/core/lists/searchbox-li-check.ts

```diff
@@ -0,0 +1,46 @@
+import { LitElement, html, css, customElement, property } from 'lit-element';
+import {nothing} from "lit-html";
+
+@customElement('searchbox-li-check')
```

Comment:
This name doesn't really make sense. The li element itself is not a "searchbox" the way the li-check element _is_ an element with a checkbox. Please refactor this.

The first option would be to see if you can compose that out of existing elements, with slight changes. If the changes are too proprietary and make usage _elsewhere_ problematic - then it should be split out.

In that case, it makes more sense to move all the "searchbox" stuff into its own widget (e.g. you'd have `searchbox-input`, `searchbox-li`, all contained under that folder - and then an example `searchbox` component)

alternatively, if the design of this element is something which is reusable in contexts other than "searchbox", please rename and express what it _is_ as opposed to where it's _used_

for example - `button-rectangle` is _not_ called `submit-button`